app/views/users/show.html.erb

```diff
@@ -5,10 +5,39 @@
 <div id='viewing-parties'>
   <div id='invited'>
     <h5>Invited</h5>
+    <% @user.invited.each do |viewing_party| %>
+      <div id="viewing_party-#{viewing_party.id}">
+      <%= image_tag "https://image.tmdb.org/t/p/w200#{viewing_party.poster_path}"%>
+      <%= link_to viewing_party.title, user_movie_path(@user, viewing_party.movie_id) %>
+      Date: <%= viewing_party.day.strftime("%B %-d, %Y") %>
+      Start time: <%= viewing_party.start_time.strftime("%-I:%M %p") %>
+      Hosted by: <%= viewing_party.user.name %>
```

Comment:
If we had time we could put this code from both Invited and Hosting divs in a partial